components/contextMenu/contextMenu.tsx

```diff
@@ -12,12 +12,23 @@ import {
 } from './style';
 import SVGIcon from '@/assets/icons/SVGIcon';
 import { labelItem, priorityItem } from '@/helpers/raw';
+import { useStores } from '@/stores';
+import { TicketDetailsInterface } from '@/utils/appTypes';
+import { updateTicketDetails } from '@/services/clientSide/ticketServices';
 
 interface Props {
   children: any;
+  ticketDetail: TicketDetailsInterface;
+  ticketIndex: number;
 }
 
-export default function CustomContextMenu({ children }: Props) {
+export default function CustomContextMenu({
```

Comment:
Use this in better way

export default function CustomContextMenu(props: Props) {
const {
  children,
  ticketDetail,
  ticketIndex,
} = props